.github/workflows/cd.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       with:
         repository: 'opensearch-project/OpenSearch'
         path: OpenSearch
-        ref: '1.0'
```

Comment:
Would you please also change in [CD](https://github.com/opensearch-project/security/blob/main/.github/workflows/cd.yml#L38)?